### PR TITLE
Manual port of #5729

### PR DESCRIPTION
**Pull request by SrNetoChan**
Goal:
Manual port of #5729

Ticket(s): #

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -83,13 +92,24 @@ And run the build from within that venv:
...
 
 .. code-block:: bash
 
  make LANG=nl html
 
+If you want to build PDFs in another language, you can use a similar syntax:
+
+.. code-block:: bash
+
+ make LANG=fr html
```
Comment (DelazJ) [inline README.rst:106]:
```suggestion
 make LANG=fr pdf
```

File: README.rst [hunk 2]
```diff
@@ -83,13 +92,24 @@ And run the build from within that venv:
...
+
+For building PDFs, you may have to install the texlive extra package for your
+specific language (e.g. ``texlive-lang-french``) or install them all
+(``texlive-lang-french``).
```
Comment (DelazJ) [inline README.rst:110]: hum, should this one be texlive-lang-all? Is that a good advice (is it not too big for what they would need it for)?
```suggestion
(``texlive-lang-all``).
```

File: README.rst [hunk 3]
```diff
@@ -83,13 +92,24 @@ And run the build from within that venv:
...
  make LANG=nl html
 
+If you want to build PDFs in another language, you can use a similar syntax:
+
+.. code-block:: bash
+
+ make LANG=fr html
+
+For building PDFs, you may have to install the texlive extra package for your
+specific language (e.g. ``texlive-lang-french``) or install them all
```
Comment (DelazJ) [inline README.rst:109]: I conclude that it works for you, right?

File: README.rst [hunk 4]
```diff
@@ -83,13 +92,24 @@ And run the build from within that venv:
...
 
  make LANG=nl html
 
+If you want to build PDFs in another language, you can use a similar syntax:
+
+.. code-block:: bash
+
+ make LANG=fr html
+
+For building PDFs, you may have to install the texlive extra package for your
```
Comment (DelazJ) [inline README.rst:108]: I need to check but I think there could be more requirements than this one. I remember I had to install years ago some tex related packages on Ubuntu in order to build our pdf. I wonder if this package includes them or it is some additional requirement due to upgrading versions

File: README.rst (hunk 2 above)
Comment (SrNetoChan) [reply README.rst:110]: 22 kb?

File: README.rst (hunk 3 above)
Comment (SrNetoChan) [reply README.rst:109]: I was able to build the Portuguese PDFs, did not try French

I will try one by one ASAP.

File: README.rst (hunk 4 above)
Comment (SrNetoChan) [reply README.rst:108]: Yes, You are right. I may be using stuff that had already installed long time ago. I plan to do some tests in a clean machine. to see how it works.

File: README.rst (hunk 2 above)
Comment (DelazJ) [reply README.rst]: You mean the all version or the french one? 
Anyway, I need to recheck what package I tried to load last day; it was many hundreds of Mo (the all version, I meant)

**Comment by SrNetoChan** [comment]
Right now I wanted to merge this to work on master, and backport to 3.10, and not the other way around.

**Pull request merged**